Orange/widgets/tests/test_domain_context_handler.py

```diff
@@ -276,6 +282,23 @@ def test_backward_compatible_params(self):
             self.assertIn(OrangeDeprecationWarning,
                           [x.category for x in w])
 
+    def test_deprecated_str_as_var(self):
+        if LooseVersion(Orange.__version__) >= LooseVersion("3.26"):
+            # pragma: no cover
+            self.fail("Remove support for variables stored as string settings "
+                      "and this test.")
```

Comment:
Should we add a comment here or open an issue about what this removal of support for string settings includes?
My wish for that list is that it contains the removal of the constant `100` from the code :)